frontend/pages/admin/UserManagementPage/_styles.scss

```diff
@@ -6,11 +6,110 @@
     padding-bottom: $pad-medium;
   }
 
-  &__sandbox-demo-message {
-    margin-top: 3.5rem;
-  }
-
   &__api-only-user {
     @include grey-badge;
   }
+
+  thead {
+    // need specificity to override datatable css
+    th {
+      &.actions__header {
+        padding-left: 0;
+      }
+      &.status__header,
+      &.role__header {
+        width: 86px; // set to prevent expanding
+      }
+    }
+  }
+
+  tbody {
+    // need specificity to override datatable css
+    td.name__cell,
+    td.role__cell,
+    td.teams__cell,
+    td.status__cell,
+    td.email__cell {
+      max-width: $col-sm;
+      white-space: nowrap;
+    }
+
+    td.status__cell,
+    td.role__cell {
+      white-space: nowrap; // Prevent No access from wrapping
+      max-width: 63px;
```

Comment:
Why is this different from the role header width?

Reply:
Removing since this actually doesn't affect the column width <3